03-attribute-operations.Rmd

```diff
@@ -554,10 +536,11 @@ world %>%
 ```
 
 ## Removing spatial information
+<!-- Shouln't that be part of chapter 2-->
```

Comment:
Since we are dealing with attribute operations in chapter 3, removing spatial information (which is a column) is very well placed here. I made this comment when I was in a rush (not a good idea) and did so because in the back of my mind I had that there was something on the geometry column in chapter 2. But we can just put a reference there to chapter 3.